src/main/java/com/github/tomakehurst/wiremock/verification/LoggedRequest.java

```diff
@@ -76,15 +77,18 @@ public LoggedRequest(String url,
 
     @JsonCreator
     public LoggedRequest(@JsonProperty("url") String url,
-                         @JsonProperty("absoluteUrl") String absoluteUrl,
-                         @JsonProperty("method") RequestMethod method,
-                         @JsonProperty("headers") HttpHeaders headers,
-                         @JsonProperty("body") String body,
-                         @JsonProperty("browserProxyRequest") boolean isBrowserProxyRequest,
-                         @JsonProperty("loggedDate") Date loggedDate) {
-        this(url, absoluteUrl, method, headers, body.getBytes(Charsets.UTF_8), isBrowserProxyRequest, loggedDate);
+    		@JsonProperty("absoluteUrl") String absoluteUrl,
+    		@JsonProperty("method") RequestMethod method,
+    		@JsonProperty("headers") HttpHeaders headers,
+    		@JsonProperty("browserProxyRequest") boolean isBrowserProxyRequest,
+    		@JsonProperty("loggedDate") Date loggedDate,
+    		@JsonProperty("body") String body,
+    		@JsonProperty("loggedDateString") String loggedDateString,
+    		@JsonProperty("bodyAsBase64") String bodyAsBase64)
```

Comment:
I had to add all of the extra JSON properties even though they are unused as test `com.github.tomakehurst.wiremock.RequestQueryAcceptanceTest.requestBodyEncodingRemainsUtf8()` was failing with `java.lang.RuntimeException at RequestQueryAcceptanceTest.java:119`. It was trying to do a base64 decode against the body, which is not base64 encoded. After plenty of digging, these extra properties got it working. I have to admit that it is a bit of a mystery why that is.